Build error: dns fails test

```
BUILD PYTHON BINDINGS
minimega.py - OK

TESTING
?       apigen  [no test files]
?       bridge  [no test files]
# dns
src/dns/example_test.go:9:2: cannot find package "github.com/miekg/dns" in any of:
        /usr/local/go/src/github.com/miekg/dns (from $GOROOT)
        /home/ubuntu/minimega/src/github.com/miekg/dns (from $GOPATH)
FAIL    dns [setup failed]
?       gonetflow       [no test files]
ok      gopacket        0.001s
```

This is a problem. Also see numerous references in the codebase such as:

https://github.com/sandia-minimega/minimega/blob/master/src/dns/dnsutil/util.go#L11

We vendorize all packages in minimega, so that the build scripts never
reach to external source. Please triage. @cdshann @jcrussell

On Thu, Aug 25, 2016 at 3:01 AM, mkunz7 <notifications@github.com> wrote:

> BUILD PYTHON BINDINGS
> minimega.py - OK
>
> TESTING
> ?       apigen  [no test files]
> ?       bridge  [no test files]
> # dns
> src/dns/example_test.go:9:2: cannot find package "github.com/miekg/dns" in any of:
>         /usr/local/go/src/github.com/miekg/dns (from $GOROOT)
>         /home/ubuntu/minimega/src/github.com/miekg/dns (from $GOPATH)
> FAIL    dns [setup failed]
> ?       gonetflow       [no test files]
> ok      gopacket        0.001s
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sandia-minimega/minimega/issues/630>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/AElrVHyxGzW27NOQD6TbS3OJRH791rO9ks5qjVnwgaJpZM4Js2En>
> .
>

--- a/src/dns/dnsutil/util.go
+++ b/src/dns/dnsutil/util.go
@@ -8,7 +8,7 @@ package dnsutil
 import (
 	"strings"
 
-	"github.com/miekg/dns"
+	"dns"
 )
 
 // AddDomain adds origin to s if s is not already a FQDN.

--- a/src/dns/example_test.go
+++ b/src/dns/example_test.go
@@ -6,7 +6,7 @@ import (
 	"log"
 	"net"
 
-	"github.com/miekg/dns"
+	"dns"
 )
 
 // Retrieve the MX records for miek.nl.

--- a/src/dns/msg_generate.go
+++ b/src/dns/msg_generate.go
@@ -47,7 +47,7 @@ func getTypeStruct(t types.Type, scope *types.Scope) (*types.Struct, bool) {
 
 func main() {
 	// Import and type-check the package
-	pkg, err := importer.Default().Import("github.com/miekg/dns")
+	pkg, err := importer.Default().Import("dns")
 	fatalIfErr(err)
 	scope := pkg.Scope()
 

--- a/src/dns/privaterr_test.go
+++ b/src/dns/privaterr_test.go
@@ -4,7 +4,7 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/miekg/dns"
+	"dns"
 )
 
 const TypeISBN uint16 = 0x0F01

--- a/src/dns/types_generate.go
+++ b/src/dns/types_generate.go
@@ -84,7 +84,7 @@ func getTypeStruct(t types.Type, scope *types.Scope) (*types.Struct, bool) {
 
 func main() {
 	// Import and type-check the package
-	pkg, err := importer.Default().Import("github.com/miekg/dns")
+	pkg, err := importer.Default().Import("dns")
 	fatalIfErr(err)
 	scope := pkg.Scope()
 

